packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer-selectors.ts

```diff
@@ -101,8 +102,8 @@ export const GeneratedAnswerSelectors: GeneratedAnswerSelector = {
     GeneratedAnswerSelectors.get().find(
       `c-quantic-radio-buttons-group [data-cy="${label}"]`
     ),
-  generatedAnswerFooter: () =>
-    GeneratedAnswerSelectors.get().find('[data-cy="generated-answer__footer"]'),
+  generatedAnswerFooterRow: () =>
+    GeneratedAnswerSelectors.get().find('.generated-answer__footer-row'),
```

Comment:
why not? best practice and more precise according to the cypress doc, plus the other selectors already use it

```suggestion
    GeneratedAnswerSelectors.get().find('[data-cy="generated-answer__footer-row"]'),
```

Reply:
Sure but I don't agree it's a best practice though. If the css class changes, I want to be notified by a potential test selector failing. If you use data-cy everywhere, you can randomly change the CSS classes of your elements and the tests still pass...